include/odp/api/spec/init.h

```diff
@@ -32,6 +32,11 @@ extern "C" {
  * ODP instance ID.
  */
 
+/**
+ * Called from signal handler
+ */
+#define ODP_TERM_FROM_SIGH	0x1u
+
```

Comment:
Two more potential flags are
a) dump stack
b)  capture trace.

I think, we can make it as HINT, so that we dont need capablity to use this or and a speicifc implemenation can ignore this if this not applicable. Some thing like,

ODP_TERM_DUMP_STACK_HINT
ODP_TERM_CAPTURE_TRACE_HINT

Reply:
Alternatively, these flags could be implementation specific, if deemed not generic enough. In any case, yes, best keep them hint-like.

Reply:
OK. In such case to avoid conflict with platform specific flasgs, introduce something like ODP_TERM_PLATFORM_TYPE_1, ODP_TERM_PLATFORM_TYPE_2, ODP_TERM_PLATFORM_TYPE_3, ODP_TERM_PLATFORM_TYPE_4.

Reply:
If API support is needed for additional flags, we could have "last flag" define in the end. However, it's trivial for implementation to select a base for all its own flags (e.g. start from bit 32), and update the base only when a new API version would overlap.

Reply:
Yes, Let add LAST_FLAG at 31 bit.

Reply:
`ODP_TERM_LAST_FLAG` in v4.